drivers/include/ds18.h

```diff
@@ -54,21 +54,20 @@ extern "C" {
 
 
 /**
- * @brief   Device descriptor for a ds18 device
+ * @brief Device initialization parameters
```

Comment:
minor nit (I know it was wrong before, too): typically we have tab-alignment/spacing between `@brief` and the description/heading (see below for the `ds18_t`).

Please amend that directly, THX!

Reply:
@smlng Do you want me to do the other @brief's in this file as well?

Reply:
ah, I see lots of inconsistencies there ... looks like we need a beautifier tool for that, too.

So leave as is, I'll withdraw my change commit - this should be done automatically by some tool